tests/integration_tests/base/base_test_post.py

```
@@ -38,9 +38,12 @@ def test_post(self, rest_client: RestClient, soap_client: SoapClient) -> None:
         """
         rest_module = self.get_module(rest_client, self.REST_MODULE_NAME)
         soap_module = self.get_module(soap_client, self.SOAP_MODULE_NAME)
-
-        rest_response = rest_module.post(self.REST_PAYLOAD)
-        soap_response = soap_module.post(self.SOAP_PAYLOAD)
+        try:
+            rest_response = rest_module.post(self.REST_PAYLOAD)
+            soap_response = soap_module.post(self.SOAP_PAYLOAD)
+        except Exception as e:
+            print(f"Error in test_post: {e.response}")
+            assert 1==2
```

Pylint (4.0.2)

[refactor] Comparison between constants: "1 == 2" has a constant value

(R0133)